Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.18-alpine as builder
+
+RUN apk add --no-cache \
+    make \
+    git \
+    bash \
+    curl \
+    findutils \
+    gcc \
+    libc-dev
+
+RUN mkdir -p /go/src/github.com/pingcap/tidb-dashboard
+WORKDIR /go/src/github.com/pingcap/tidb-dashboard
+
+# Cache dependencies
+COPY go.mod .
+COPY go.sum .
+
+RUN GO111MODULE=on go mod download
+
+COPY . .
+
+RUN make server
```

Comment:
I think we should not just compile the backend, but to embed the frontend asset as well?

Reply:
`make ui && UI=1 make server` can build the frontend and embed it to go binary.